docs/pages/installation/kinD.md

```diff
@@ -153,9 +156,9 @@ Run the following:
 
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- $ git clone https://github.com/layer5io/meshery.git; cd meshery
- $ kubectl create namespace meshery
- $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
+$ kubectl create ns meshery
+$ helm repo add meshery https://meshery.io/charts/
+$ helm install meshery meshery/meshery -n meshery

```

Comment:
```suggestion
$ helm repo add meshery https://meshery.io/charts/
$ helm install meshery meshery/meshery -n meshery --create-namespace
```
Since the values of namespace are fixed on Helm charts, the above flag will suffice to create a namespace in cluster